### PR TITLE
Refactor/list layout

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-5369
| License          | BSD 3-Clause

## Description
Refactor extracted from https://github.com/ovh/manager/pull/3330

## Related
Aggregation of reviewed PR:  
- [x] https://github.com/ovh/manager/pull/3277
- [x] https://github.com/ovh/manager/pull/3324
- [x] https://github.com/ovh/manager/pull/3359
- [x] https://github.com/ovh/manager/pull/3395
- [x] https://github.com/ovh/manager/pull/3697/commits/ebbf940bd5b46b22663e3b46a6554dad20bc337e

### CODE REVIEW

**Review by antleblanc — CHANGES_REQUESTED** [review]
@marie-j In order to be able to merge this Pull Request, could you please resolve existing conflict?

Thank you!

File: packages/manager/apps/hub/src/dashboard/products-list/config.js [hunk 1]
```diff
@@ -1,61 +1,15 @@
-import { find, get, head, isEmpty, map, startCase } from 'lodash-es';
-
+import { find, get, head } from 'lodash-es';
```
Comment (Heiiji) [inline packages/manager/apps/hub/src/dashboard/products-list/config.js:1]: Import lodash `find` is necessary ? (i've impression, native one is used here)

Comment (marie-j) [reply packages/manager/apps/hub/src/dashboard/products-list/config.js:1]: Used here: https://github.com/ovh/manager/blob/dc00d786c263e49976c0db41d20a98196d3731cc/packages/manager/apps/hub/src/dashboard/products-list/config.js#L17

File: packages/manager/modules/ng-layout-helpers/src/list/list-layout.controller.js [hunk 2]
```diff
@@ -39,8 +38,19 @@ export default class ListLayoutCtrl {
     return this.sort === property ? this.sortOrder.toLowerCase() : '';
   }
 
+  getDisplayedColumns(columns) {
+    this.displayedColumns = columns
+      ? JSON.stringify(
+          map(
```
Comment (Heiiji) [inline packages/manager/modules/ng-layout-helpers/src/list/list-layout.controller.js:44]: Native map is used before, is this usefull to import Lodash one for this one ? perf issue ?

File: packages/manager/apps/hub/src/dashboard/products-list/config.js (hunk 1 above)
Comment (marie-j) [reply packages/manager/apps/hub/src/dashboard/products-list/config.js:1]: USed here: https://github.com/ovh/manager/blob/dc00d786c263e49976c0db41d20a98196d3731cc/packages/manager/apps/hub/src/dashboard/products-list/config.js#L17

File: packages/manager/modules/ng-layout-helpers/src/list/list-layout.utils.js [hunk 3]
```diff
@@ -0,0 +1,355 @@
+import get from 'lodash/get';
```
Comment (Heiiji) [inline packages/manager/modules/ng-layout-helpers/src/list/list-layout.utils.js:1]: i've seen before few lodash module import in one line with lodash-es, is this better to do like here or with 1 import ?

File: packages/manager/modules/ng-layout-helpers/src/list/list-layout.controller.js (hunk 2 above)
Comment (marie-j) [reply packages/manager/modules/ng-layout-helpers/src/list/list-layout.controller.js:44]: Lodash handles more cases like if the parameter is undefined and stuff like that

**Review by Heiiji — DISMISSED** [review]

File: packages/manager/modules/ng-layout-helpers/src/list/list-layout.utils.js (hunk 3 above)
Comment (marie-j) [reply packages/manager/modules/ng-layout-helpers/src/list/list-layout.utils.js:1]: Yes `lodash-es` is supposed to be better in terms of performance but refactoring to use it everywhere should be done in a separate PR as it is a separate concern

**Comment by marie-j** [comment]
Rebase to fix conflicts

**Review by antleblanc — APPROVED** [review]

**Pull request merged**